CHANGELOG.md

```diff
@@ -11,6 +11,29 @@ and this project **only** adheres to the following _(as defined at [Semantic Ver
 > - MINOR version when you add functionality in a backward compatible manner
 > - PATCH version when you make backward compatible bug fixes
 
+## [v3.1.1] - 2025-06-09
+
+This patch release resolves bugs related to connection, `ips` tracking, and dependencies.
+
+### Changed
+
+- Set compatibility to iRODS 5.0.1 (#695).
```

Comment:
Remove this if https://github.com/irods/python-irodsclient/pull/738/files#r2132704435 is dropped.

Reply:
Removed.